### PR TITLE
Updated to clarify cloud mailbox requirements

### PR DESCRIPTION
**Pull request by RossAdams**
Add details that some settings require a cloud mailbox

### CODE REVIEW

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/security/office-365-security/configure-the-outbound-spam-policy.md [hunk 1]
```diff
@@ -137,6 +137,9 @@ Creating a custom outbound spam policy in the Security & Compliance Center creat
 
 5. (Optional) Expand the **Recipient Limits** section to configure the limits and actions for suspicious outbound email messages:
 ]
+     > [!NOTE]
+     > These settings are only applicable to cloud based mailboxes.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/configure-the-outbound-spam-policy.md]:
```suggestion
     > These settings are only applicable to cloud-based mailboxes.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor edit.

**Comment by mypil** [comment]
@RossAdams  - please merge the commit on the copy/edit review done by JohanFreelancer9 or make the necessary changes on the PR following the suggestions made. Thank you.

**Pull request merged**